R/f7Grid.R

```diff
@@ -82,7 +82,7 @@ f7Col <- function(..., width = NULL, resizable = FALSE){
 
   # width
   if(!is.null(width))
-    cl <- paste0(col, width)
+    cl <- paste0(cl, "-", width)
 
   # sizable
   if(resizable) cl <- paste(cl, "resizable")
```

Comment:
Below I would also do:
```r
if (resizable) col <- shiny::tagAppendAttributes(col, style = sprintf("min-width: %s", param))
```
to consider the minimum allowed width.

Reply:
Can you add an extra parameter to account for fixed-width columns? We need to add the class "resizable-fixed" after "resizable" if fixed is TRUE